### ISSUE TITLE
[Xamarin.Forms] [Android] Picker gets a permanent focus when setting gravity

### ISSUE THREAD

**Issue by yuv4ik** [xamarin/Xamarin.Forms#4380]
### Description
Creating a custom renderer for a `Picker` control just to set a gravity results in a permanent focus and its consequences.

### Steps to Reproduce

1. Add a `Picker` control to the main page:
```
<Picker
    Grid.Row="0"
    Title="Tap me once and then the hardware volume buttons">
    <Picker.Items>
        <x:String>a</x:String>
    </Picker.Items>
</Picker>
```
2. Add and register a custom renderer:
```
[assembly: ExportRenderer(typeof(Picker), typeof(GenericPickerRenderer))]
namespace XFAndroidPickerFocusBug.Droid
{
    public class GenericPickerRenderer : PickerRenderer
    {
        public GenericPickerRenderer(Context context) : base(context) { }

        protected override void OnElementChanged(ElementChangedEventArgs<Picker> e)
        {
            base.OnElementChanged(e);
            if (Control != null)
            {
                Control.Gravity = Android.Views.GravityFlags.Center;
            }
        }
    }
}
```

### Expected Behavior
The `Picker` should loose the focus after user interaction: successful selection or cancellation.

### Actual Behavior
The `Picker` is staying focused permanently and pressing the hardware volume up button for example results in showing the selection popup / alert view twice. Please check the attached gif.

### Basic Information

- Version with issue: 3.3.0.967583
- IDE: Visual Studio for Mac
- Platform Target Frameworks: 
  - Android: 8.1
- Android Support Library Version: V7
- Affected Devices: emulator / Samsung galaxy s5 / Mi MiX 2

### Screenshots

![Demo](https://github.com/yuv4ik/XFAndroidPickerFocusBug/raw/master/Screenshots/xf_android_picker_bug.gif)

### Reproduction Link

https://github.com/yuv4ik/XFAndroidPickerFocusBug

**Comment by kingces95** [comment]
Filing custom renderer bugs as enhancements.

**Comment by hnabbasi** [comment]
It does that even without a custom renderer on a device on API 23. And if you get focus in code as `Picker.Focus()`, it freaks out if you have more than one picker on a page on API 26.

**Comment by dhruvcom** [comment]
I was experiencing the same issue when I updated my XF version to 3.4. Picker dialog opened up twice. To remove that - in custom renderer I added this :

**Comment by aleksandrsmyk** [comment]
@dhruvcom Thank you! It helps me. I faced the same issue with 3.4 XF.

**Comment by jfversluis** [comment]
I assume this got fixed also by #6961
